config/prismjs/dvctable.js

```diff
@@ -22,39 +19,8 @@ const boldAndItalicConfig = {
 }
 
 Prism.languages.dvctable = {
-  'bg-white': {
-    pattern: getTableCellBgColorRegex('(white|neutral)'),
-    inside: {
-      hide: {
-        pattern: /(white|neutral):/
-      },
-      ...boldAndItalicConfig
-    }
-  },
-  'bg-yellow': {
-    pattern: getTableCellBgColorRegex('(yellow|metric)'),
-    inside: {
-      hide: {
-        pattern: /(yellow|metric):/
-      },
-      ...boldAndItalicConfig
-    }
-  },
-  'bg-blue': {
-    pattern: getTableCellBgColorRegex('(blue|param)'),
-    inside: {
-      hide: {
-        pattern: /(blue|param):/
-      },
-      ...boldAndItalicConfig
-    }
-  },
-  ...boldAndItalicConfig
-}
-
-Prism.languages.dvctablehorizontals = {
   rows: {
-    pattern: /((?<=^|\n)\s[^─][\s\S]*?(:?\n|$))+/,
+    pattern: /((?<=^|\n)[^─]{2}[\s\S]*?(:?\n|$))+/,
```

Comment:
I had to update the regex to allow this blog table to work correctly:

![image](https://user-images.githubusercontent.com/43496356/149014157-2d36475a-edc5-46f6-8d79-cddb89694813.png)

Reply:
Hmmm so just to support splitting tables with `...`? Maybe we should remove the `...` or the second half of that table instead.

Reply:
What's the blog post URL? Thanks

Reply:
It's from `/blog/transfer-learning-experiments`. But this change may be useful to keep, since it essentially stops a table row from having to have white space as its first character.